wagtail/admin/tests/test_templatetags.py

```diff
@@ -1068,3 +1068,54 @@ def test_icon_name(self):
         self.assertEqual(len(invalid_icons), 0)
         icon = soup.select_one("ol li:last-child svg use[href='#icon-site']")
         self.assertIsNotNone(icon)
+
+
+class DarkLightModeTest(AdminTemplateTestUtils, WagtailTestUtils, TestCase):
+    fixtures = ["test.json"]
+
+    def setUp(self):
+        self.request = get_dummy_request()
+        self.user = self.login()
+        self.request.user = self.user
+
+        self.profile, created = UserProfile.objects.get_or_create(user=self.user)
+
+    def test_dark_mode(self):
+        self.profile.theme = "dark"
+        self.profile.save()
+
+        template = """
+            {% load wagtailadmin_tags %}
+            <body class="{% admin_theme_color_scheme %}">
+                {% page_breadcrumbs page 'wagtailadmin_explore' %}
```

Comment:
I recommend we remove the breadcrumbs template tag usage, it's unrelated to this unit test. 

```suggestion
```